src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/session/SessionTemplate.kt

```diff
@@ -33,6 +37,14 @@ data class SessionTemplate(
   @Column(nullable = false)
   val visitRoom: String,
 
+  @ManyToMany(fetch = FetchType.LAZY, cascade = [CascadeType.ALL])
```

Comment:
Sorry we did discuss this yesterday but do you think OneToMany would be better here and have PermittedSessionLocation as a child table since we do not have the full list of Session locations and only add it adhoc based on the SessionTemplate. So it would be similar to a person adding the allowed location as text for a session template. Happy to discuss further.

Reply:
It's a many to many, will save us a lot of time especially since we will have lot's of things to config,... it has it's down side but should be manage via font end,... will think about this tonight and have a chat tomorrow